src/base/bittorrent/dbresumedatastorage.cpp

```diff
@@ -128,6 +131,25 @@ namespace
                 .arg(quoted(tableName), jointNames, jointValues);
     }
 
+    QString makeUpdateStatement(const QString &tableName, const QVector<Column> &columns)
+    {
+        QStringList names;
```

Comment:
If this function is commonly used and performance-related, then it might be wise to append to `QString` directly and drop the last `','` manually.

Reply:
> If this function is commonly used and performance-related, then it might be wise to append to `QString` directly and drop the last `','` manually.

Are you sure that it will be so much more optimal?

Reply:
>Are you sure that it will be so much more optimal?

It depends on whether there are many elements in `columns`.
However if the coding complexity is manageable, I would go for it anyway.

Reply:
>However if the coding complexity is manageable, I would go for it anyway.

Done.